### PR TITLE
contact for public and admin (read, add, delete)

### PR DESCRIPTION
**Pull request by arpelayo**

### CODE REVIEW

File: php/application/views/layout.php [hunk 1]
```diff
@@ -53,10 +53,12 @@
 		      {{/isLoggedIn}}
           <li><a href="<?= base_url('/about'); ?>">About Project Tree Trail</a></li>
           <li><a href="#" id="badge-filter">Filter Badges</a></li>
+          <li><a href="<?= base_url('/contact'); ?>">Contact</a></li>
```
Comment (fskreuz) [inline php/application/views/layout.php]: Duha ka link?

File: php/application/views/layout.php [hunk 2]
```diff
@@ -53,10 +53,12 @@
...
         </ul>
         <ul class="nav navbar-nav navbar-right">
       		{{#isLoggedIn}}
       		<li><a href="<?= base_url('/dashboard'); ?>">Administrator Dashboard</a></li>
+          <li><a href="<?= base_url('/contact'); ?>">Contact</a></li>
```
Comment (fskreuz) [inline php/application/views/layout.php]: ?

Comment (arpelayo) [inline php/application/views/layout.php]: murag manage contact ni sa admin

File: php/application/views/layout.php (hunk 1 above)
Comment (arpelayo) [inline php/application/views/layout.php]: diba sa public ni?

Comment (fskreuz) [inline php/application/views/layout.php]: Pero duha "Contact" naa sa navigation mugawas?

Comment (arpelayo) [inline php/application/views/layout.php]: hmmm sa ako local copy usa ra nigawas.. di ko ka test for admin and super-admin :<

Comment (fskreuz) [inline php/application/views/layout.php]: :+1: We'll wait for the bug report lng. :trollface:

**Pull request merged**